### DISCUSSION TITLE
Thumbnail images not working properly

### DISCUSSION THREAD

**Discussion by nibinglobal** [breezefront/community#21] [1. Help]
Whenever I turn off the Breeze module theme setting from "Use theme settings" to "No" I get this error.
Look at this screenshot.
![Screenshot 2022-10-08 at 9 39 09 AM](https://user-images.githubusercontent.com/78966425/194687645-cd591558-f448-4faf-![Screenshot 2022-10-08 at 9 44 12 AM](https://user-images.githubusercontent.com/78966425/194687651-e3103eb0-707c-4ed8-a45a-5cd7599ac248.png)

**Comment by nibinglobal** [comment]
![Screenshot 2022-10-08 at 9 39 09 AM](https://user-images.githubusercontent.com/78966425/194687712-d273fa15-a3dc-432b-bedf-2b0c97c0e7c9.png)

**Comment by vovayatsyuk** [comment]
It happens because when breeze js is disabled default gallery is rendered, but the theme doesn’t has styles for it.

It’s not a bug since we do not officially support disabled mode for breeze themes. This config was made for luma based themes only.
